app/controllers/api/v2/permissions_controller.rb

```diff
@@ -34,6 +35,14 @@ def resource_types
         @total = @resource_types.size
         render :resource_types, :layout => 'api/v2/layouts/index_layout'
       end
+
+      private
+
+      def parameter_deprecation
```

Comment:
@ehelms, searching based on resource type is used in filter form, see https://github.com/theforeman/foreman/blob/develop/app/views/filters/_form.html.erb#L28 and https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/filters.js#L6 so I don't think this should be deprecated, I would just update the else branch